### DISCUSSION TITLE
Vendor Qualification of GitHub

### DISCUSSION THREAD

**Discussion by jasonhinch** [community/community#137044] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Has anyone had to do a vendor qualification of GitHub (questions on functionality, audit trail etc)? How did you do this without being able to contact GitHub directly? Or is there an email contact address I could use for this? Tx :D

**Comment by Yackens** [comment]
GitHub provides detailed documentation for [GitHub Enterprise](https://docs.github.com/en/enterprise-cloud@latest), which includes information on audit logs, security, and other features that might be relevant for vendor qualification. I advice you to check it there. 

From my understanding, sorry to tell you but you will need to contact GitHub to get the vendor qualifications that you need. You can contact their sales team directly through the [GitHub Enterprise Sales Contact](https://github.com/enterprise/contact). They can often provide detailed information as part of the sales process.

If you found this information useful, don't forget to mark it as the accepted answer ✅.
